README.md

```diff
@@ -86,4 +87,24 @@ Log.ForContext("CustomGroupKeyProperty", "TransactionId-12345").Error(new Except
 ```csharp
 Log.ForContext("CustomTagsProperty", new[] {"tag1", "tag2"}).Error(new Exception("random error"), "other information");
 Log.Error(new Exception("random error"), "other information {@CustomTagsProperty}", new[] {"tag3", "tag4"});
+```
+
+#### userInfoProperty
+`type: string`
+
+`default: null`
+
+This is null by default, so you need to configure the userInfoProperty name if you want to log more user information in this way. This will cause the RaygunIdentifierMessage to be included in the "User" section of the Raygun payload, allowing the information to be picked up by the "Users" section of the Raygun service. This will not happen if the RaygunIdentifierMessage is destructured into the log message. Sending user information in this way will overwrite the use of the userNameProperty.
+
+The user identifier passed into the RaygunIdentifierMessage constructor could be the users name, email address, database id or whatever works best for you to identify unique users.
+
+```csharp
+var userInfo = new RaygunIdentifierMessage("12345")
+{
+    FirstName = "John",
+    FullName = "John Doe",
+    Email = "johndoe@email.address"
+};
+
+Log.ForContext("CustomUserInfoProperty", userInfo).Error(new Exception("random error"), "other information");
```

Comment:
It would be more idiomatic here to pass `destructureObjects: true`:

```csharp
Log.ForContext("CustomUserInfoProperty", userInfo, true)
```

This would get away from needing parsing code later in the logging pipeline, and have the benefit of presenting structured data to all sinks attached to the pipeline.

With `destructureObjects: true`, the `CustomUserInfoProperty` property will have a value of type `StructureValue`, and its members will be `FirstName`, `FullName`, etc.

Reply:
Thanks for pointing this out, I've added support for destructured values. I was working with providing the RaygunIdentifierMessage from a properties dictionary, not in the log message, and didn't realize you could specify values in a dictionary to be destructured.

I would like to keep the support for parsing non-destructured values, so that this feature just works regardless. I have read that destructuring may be preferred to be avoided when using Serilog within Microsoft.Extensions.Logging for example as it brings Serilog concepts up into the generic logging code.